spec/support/feature_helpers.rb

```diff
@@ -1,33 +1,9 @@
 module FeatureHelpers
-  def answer_all_student_loans_claim_questions
-    start_claim
-    choose_school schools(:penistone_grammar_school)
-    choose_subjects_taught
-    choose_still_teaching "Yes, at another school"
-    choose_school schools(:hampstead_school)
-    choose_leadership
-    click_on "Continue"
-
-    perform_verify_step
-    click_on "Continue"
-
-    fill_in :claim_teacher_reference_number, with: "1234567"
-    click_on "Continue"
-
-    fill_in "National Insurance number", with: "QQ123456C"
-    click_on "Continue"
-
-    answer_student_loan_plan_questions
-    fill_in I18n.t("student_loans.questions.student_loan_amount", claim_school_name: claim.eligibility.claim_school_name), with: "1100"
-    click_on "Continue"
-
-    fill_in I18n.t("questions.email_address"), with: "name@example.tld"
-    click_on "Continue"
-
-    fill_in "Name on the account", with: "Jo Bloggs"
-    fill_in "Sort code", with: "123456"
-    fill_in "Account number", with: "87654321"
-    click_on "Continue"
+  def submittable_claim
```

Comment:
Could we have used the `:submittable` factory instead?

Reply:
That was my inital thought, but we need the claim to be present in the session, so we need to start a claim, and then update that claim. I've got a better solution coming now though, thanks to a prompt from @robbiepaul

Reply:
I also ran a quick test to see if there was any improvements using a factory then doing `allow_any_instance_of(ClaimController).to receive(:current_claim) { claim }` 

I know we're not keen on stubbing controller methods on feature specs but I was curious if there would be a performance boost, but there was none over the proposed implementation 👍

Reply:
Ah, that's good to know 👍 I do prefer staying as close as possible reality in feature specs, so it's nice that we're not slowing anything down too much doing it the right way

Reply:
Gotya! Thanks!